tracpro/polls/tasks.py

```diff
@@ -48,8 +47,8 @@ def org_task(self, org):
 
         total_runs = 0
         for poll in Poll.objects.active().by_org(org):
-            poll_runs = client.get_runs(flows=[poll.flow_uuid], after=last_time, before=until)
-            total_runs += len(poll_runs)
+            poll_runs = client.get_runs(flow=poll.flow_uuid, after=last_time, before=until)
+            total_runs += len(poll_runs.all())
```

Comment:
I'm not sure about this `.all()` - do we need it here?  Our client makes the return value from client calls iterable without calling `.all()`.